sdcm/cluster.py

```diff
@@ -3045,6 +3049,45 @@ def reload_config(self):
             return
         self.remoter.run(f"sudo kill -s SIGHUP {pid}")
 
+    def get_raft_status(self) -> str:
+        """ get raft state
+
+        If execute query, raft state will be return if raft is enabled,
+        otherwise result will be empty
+        Example
+
+            key                  | value
+            ----------------------+--------------------------
+            group0_upgrade_state | use_post_raft_procedures
+
+            (1 rows)
+
+
+        """
+        result = self.run_cqlsh("SELECT * FROM system.scylla_local WHERE key = 'group0_upgrade_state'",
```

Comment:
can we switch to python-driver? So it won't be dependent on cqlsh ?

Reply:
yes, i will do